app/src/js/utils/table-config/rules.js

```diff
@@ -51,39 +44,4 @@ export const tableColumns = [
   }
 ];
 
-const removeEsFields = (data) => {
-  const { queriedAt, timestamp, stats, ...nonEsFields } = data;
-  return nonEsFields;
-};
-
-export const bulkActions = (rules) => [{
```

Comment:
Why are these removed?

Reply:
I couldn't use the React Router hooks in this utility because it wasn't a React component so I moved them to the Rules `overview.js` component (similar another component that was already set up this way previously). Sorry for the alarm.